iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/bicep/ExpressionImplTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.arm.tree.impl.bicep;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.sonar.iac.arm.parser.BicepParser;
+import org.sonar.iac.arm.parser.bicep.BicepLexicalGrammar;
+import org.sonar.iac.arm.tree.api.ArmTree;
+import org.sonar.iac.arm.tree.api.StringLiteral;
+import org.sonar.iac.arm.tree.api.bicep.SyntaxToken;
+import org.sonar.iac.common.extension.ParseException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.sonar.iac.common.testing.IacTestUtils.code;
+
+class ExpressionImplTest {
+
+  BicepParser parser = BicepParser.create(BicepLexicalGrammar.EXPRESSION);
```

Comment:
Maybe you can be inspired by this [implementation](https://github.com/SonarSource/sonar-iac/blob/a127afb8f67b3168c568e87fc57214b5f66823bb/iac-extensions/terraform/src/test/java/org/sonar/iac/terraform/tree/impl/TerraformTreeModelTest.java#L43) of a test framework for Terraform

Reply:
Ticket [SONARIAC-937](https://sonarsource.atlassian.net/browse/SONARIAC-937) created for this.